src/cpp/session/modules/SessionFindTests.cpp

```diff
@@ -42,11 +42,16 @@ const size_t rMatchOff = 10;
 const size_t caseMatchOn = 21;
 const size_t caseMatchOff = 27;
 
-const std::string kFindRegex("\\([a-z]\\)\\1\\{2\\}\\([a-z]\\)\\2\\{2\\}");
+const std::string kFindRegex("([a-z])\\1{2}([a-z])\\2{2}");
```

Comment:
Were the original escaping patterns here not actually needed or do your changes affect how the user needs to enter their queries? If the ladder, can you include the differences in the description of this PR?

Reply:
Yes, I think that with ERE enabled, and the regex option selected, the user does not need to escape parentheses and curly braces. If they want a literal parenthesis or curly brace, or if they want a literal `?` or `+`, they will now escape it. Will do!